src/rules/alt-text.js

```diff
@@ -39,7 +39,7 @@ const ariaLabelHasValue = (prop) => {
   if (value === undefined) {
     return false;
   }
-  if (typeof value === 'string' && value.length === 0) {
+  if (typeof value === 'string' && value.length === 0 && value === '') {
```

Comment:
`value === ''` and `value.length === 0`, when it's already a string, are the same condition.
```suggestion
  if (typeof value === 'string' && value.length === 0) {
```